back/engines/free/email_campaigns/app/views/email_campaigns/moderator_digest_mailer/campaign_mail.mjml

```diff
@@ -1,21 +1,22 @@
+<mj-section
+  padding="0 25px"
+>
+  <mj-column>
+    <mj-text>
+      <h2 style="padding: 20px 0">
+        <%= event.project_name %>
+      </h2>
+    </mj-text>
+  </mj-column>
+</mj-section>
 
-<mj-wrapper background-color="#F6F6F6" full-width="full-width" padding="40px 0 0">
-
-  <mj-section>
-    <mj-column>
-      <mj-text>
-        <h2 style="padding: 20px 0">
-          <%= event.project_name %>
-        </h2>
-      </mj-text>
-    </mj-column>
-  </mj-section>
-
-  <!-- 3 statistics box-->
+<!-- 3 statistics box-->
+<mj-wrapper
+  padding="0 25px"
+>
   <mj-section
-    background-color="#FFFFFF"
-    padding="40px 25px"
     border="1px solid #EAEAEA"
+    padding="25px"
     border-radius="5px"
   >
```

Comment:
So this is the trick to have a bordered section be the same width as other sections like I explained in the calll.